src/ProjectFileManager.h

```diff
@@ -91,7 +81,14 @@ class AUDACITY_DLL_API ProjectFileManager final
 
    static bool IsAlreadyOpen(const FilePath &projPathName);
 
-   void OpenFile(const FilePath &fileName, bool addtohistory = true);
+   /*!
+    Opens files of many kinds.  In case of import (sound, MIDI, or .aup), the undo history is pushed.
+    @param fileName the name and contents are examined to decide a type and open appropriately
+    @param addtohistory whether to add .aup3 files to the MRU list (but always done for imports)
+    @return if something was successfully opened, the project containing it; else null
+    */
```

Comment:
Do we have a policy on whether to add parameter comments in the declaration or the definition?  I was looking for the comment about the return type in the .cpp.

Reply:
My preference is to put it in a header file, because that is the place to document interface sufficient to understand usage.  The .cpp file is for implementation, not interface.